android/src/main/java/org/reactnative/camera/RNCameraView.java

```diff
@@ -261,6 +256,9 @@ private void initBarcodeReader() {
   }
 
   public void setShouldScanBarCodes(boolean shouldScanBarCodes) {
+    if (shouldScanBarCodes && !mBarcodeDetector) {
```

Comment:
I think this is Java, not JavaScript. How does this compile? 
I mean “!mBarcodeDetector” part. It can’t be converted to boolean automatically, can it?

Reply:
Yup, should be:
`if (shouldScanBarCodes && mBarcodeDetector == null) {`